src/client/pythonEnvironments/info/interpreter.ts

```diff
@@ -73,3 +74,7 @@ export async function getInterpreterInfo(
     }
     return extractInterpreterInfo(python.pythonExecutable, json);
 }
+
+export function getInterpreterHash(interpreter: PythonEnvironment | {path: string}){
```

Comment:
Consistent way to get a unique identifier for a python interpreter

Reply:
Is path always enough? Meaning it is the single thing that represents an interpreter?